addons/explosives/initKeybinds.inc.sqf

```diff
@@ -0,0 +1,51 @@
+#include "\a3\ui_f\hpp\defineDIKCodes.inc"
+
+["ACE Explosives", QGVAR(openCellphone), LLSTRING(cellphone_displayName),
+{
+    if (
+        !('ACE_Cellphone' in ([ACE_player] call EFUNC(common,uniqueItems))) ||
+        !([ACE_player, objNull, ["isNotSwimming", "isNotInside", "isNotSitting"]] call EFUNC(common,canInteractWith))
+    ) exitWith {};
+    closeDialog 0;
+    createDialog 'Rsc_ACE_PhoneInterface';
+},
+{false},
+[DIK_C, [false, false, true]], false, 0] call CBA_fnc_addKeybind; // (ALT+C)
+
+["ACE Explosives", QGVAR(detonateActiveClacker), LLSTRING(DetonateAllOnActive),
+{
+    // Prevent use of keybind while surrendering or captive
+    if !([ACE_player, objNull, ["isNotSwimming", "isNotInside", "isNotSitting"]] call EFUNC(common,canInteractWith)) exitWith {};
+
+    private _detonator = GVAR(activeTrigger);
+    if (_detonator == "" || !(_detonator in ([ACE_player] call FUNC(getDetonators)))) exitWith {};
+
+    // When using a Dead Man's Switch, skip all other logic and just call fnc_onIncapacitated, since it already handles everything that is required to detonate all connected explosives
+    if (_detonator == "ACE_DeadManSwitch") exitWith {
+        [ACE_player] call FUNC(onIncapacitated);
+    };
+
+    private _range = getNumber (configFile >> "CfgWeapons" >> _detonator >> QGVAR(Range));
+
+    private _explosivesList = [];
+    {
+        if (!isNull (_x select 0)) then {
+            private _required = getArray (configFile >> "ACE_Triggers" >> _x select 4 >> "requires");
+            if (_detonator in _required) then {
+                _explosivesList pushBack _x;
+            };
+        };
+    } forEach ([ACE_player] call FUNC(getPlacedExplosives));
+
+    [ACE_player, _range, _explosivesList, _detonator] call FUNC(detonateExplosiveAll);
+},
+{false},
+[DIK_C, [false, true, true]], false, 0] call CBA_fnc_addKeybind; // (CTRL+ALT+C)
```

Comment:
Which way is standard? It appears to be 50/50 when looking at all `CBA_fnc_addKeybind;` comments in the codebase.

Reply:
Currently it's all different types. I was hoping to standardise on the style I suggest here, but I haven't asked the input of the other ACE members. For now this can be ignored though, I'll make a PR to address all of them at once.